src/bci_build/package/node.py

```diff
@@ -24,10 +24,7 @@ def _get_node_kwargs(ver: Literal[16, 18, 20], os_version: OsVersion):
     return {
         "name": "nodejs",
         "os_version": os_version,
-        "is_latest": (
-            (ver == 18 and os_version == OsVersion.SP5)
-            or (ver == 20 and os_version == OsVersion.TUMBLEWEED)
-        ),
+        "is_latest": ver == 20,
```

Comment:
that will add a "latest" entry for the SP6 containers.. I think we only want it on SP5 currently? why not simply change the `ver ==18 and os_version == OsVersion.SP5` to `ver == 20 and os_version == OsVersion.SP5` ?

Reply:
Because we have nodejs20 in sp5 now and would like to have that one as `latest` too. I've amended the commit to fix this.